parcels/fieldset.py

```diff
@@ -426,62 +482,76 @@ def from_nemo(cls, filenames, variables, dimensions, indices=None, mesh='spheric
-
-               1. spherical (default): Lat and lon in degree, with a
-                  correction for zonal velocity U near the poles.
-               2. flat: No conversion, lat/lon are assumed to be in m.
-        :param allow_time_extrapolation: boolean whether to allow for extrapolation
-               (i.e. beyond the last available time snapshot)
-               Default is False if dimensions includes time, else True
-        :param time_periodic: To loop periodically over the time component of the Field. It is set to either False or the length of the period (either float in seconds or datetime.timedelta object). (Default: False)
-               This flag overrides the allow_time_interpolation and sets it to False
-        :param tracer_interp_method: Method for interpolation of tracer fields. It is recommended to use 'cgrid_tracer' (default)
-               Note that in the case of from_nemo() and from_cgrid(), the velocity fields are default to 'cgrid_velocity'
-        :param chunksize: size of the chunks in dask loading. Default is None (no chunking)
+        Parameters
+        ----------
+        filenames :
+            Dictionary mapping variables to file(s). The
+            filepath may contain wildcards to indicate multiple files,
+            or be a list of file.
+            filenames can be a list [files], a dictionary {var:[files]},
+            a dictionary {dim:[files]} (if lon, lat, depth and/or data not stored in same files as data),
+            or a dictionary of dictionaries {var:{dim:[files]}}
+            time values are in filenames[data]
+        variables : dict
+            Dictionary mapping variables to variable names in the netCDF file(s).
+            Note that the built-in Advection kernels assume that U and V are in m/s
+        dimensions : dict
+            Dictionary mapping data dimensions (lon,
+            lat, depth, time, data) to dimensions in the netCF file(s).
+            Note that dimensions can also be a dictionary of dictionaries if
+            dimension names are different for each variable.
+            Watch out: NEMO is discretised on a C-grid:
+            U and V velocities are not located on the same nodes (see https://www.nemo-ocean.eu/doc/node19.html ).
+
+            +-----------------------------+-----------------------------+-----------------------------+
+            |                             |         V[k,j+1,i+1]        |                             |
+            +-----------------------------+-----------------------------+-----------------------------+
+            |U[k,j+1,i]                   |W[k:k+2,j+1,i+1],T[k,j+1,i+1]|U[k,j+1,i+1]                 |
+            +-----------------------------+-----------------------------+-----------------------------+
+            |                             |         V[k,j,i+1]          +                             |
+            +-----------------------------+-----------------------------+-----------------------------+
```

Comment:
These tables don't show as nice as they do in the old version? Compare https://veckoparcels.readthedocs.io/en/docstrings/#parcels.fieldset.FieldSet.from_c_grid_dataset to https://parcels.readthedocs.io/en/latest/#parcels.fieldset.FieldSet.from_c_grid_dataset